servus/serializable.h

```diff
@@ -99,23 +100,28 @@ class Serializable
 #endif
 
     /**
-     * Set a new function called after the object has been updated.
-     *
-     * @return the previously set function.
+     * Set a new function called after every change on the local object
+     * instance (e.g. Collage distributable).
      */
-    SERVUS_API ChangeFunc setUpdatedFunction( const ChangeFunc& func );
+    SERVUS_API void setLocalChangeFunction( const ChangeFunc& func );
+
+    /**
+     * Set a new function called after the object has been updated remotely
+     * (via a subscriber, a http server, loading from file...).
+     */
+    SERVUS_API void setRemoteUpdateFunction( const ChangeFunc& func );
```

Comment:
I'm fine with Raphael's propositions, also because I can't come up with better ones :)
For more API safety, the three functions should be different types. Today they have the same signature, but for instance the changed callback could hint which member has changed. This change won't break then the other two functions.